common/src/state/identity.rs

```diff
@@ -59,6 +62,10 @@ impl Identity {
     pub fn platform(&self) -> Platform {
         self.platform
     }
+
+    pub fn did(&self) -> DID {
```

Comment:
This shouldnt be needed since the `Identity` wrapper impl `Deref`, which would give you access to `Identity::did_key` from `warp::multipass::identity::Identity`